src/WooCommerce.php

```diff
@@ -448,7 +469,7 @@ public static function woocommerce_sale_flash($output, $post, $product) {
     else {
       $sale_percentage = get_post_meta($product->get_id(), '_sale_percentage', TRUE);
     }
-    $minimum_sale_percentage = get_option('_minimum_sale_percentage_to_display_label', 10);
+    $minimum_sale_percentage = get_option('_minimum_sale_percentage_to_display_label', SALE_BUBBLE_MIN_AMOUNT);
```

Comment:
As this is a class constant it should be `static::SALE_BUBBLE_MIN_AMOUNT`
```suggestion
    $minimum_sale_percentage = get_option('_minimum_sale_percentage_to_display_label', static::SALE_BUBBLE_MIN_AMOUNT);
```

Reply:
Ah! Anyway I will fix that.